src/Orchard.Web/Modules/Orchard.Email/Services/EmailMessagingChannel.cs

```diff
@@ -53,7 +53,7 @@ public class EmailMessagingChannel : IMessagingChannel {
                 smtpClient.EnableSsl = smtpSettings.EnableSsl;
                 smtpClient.DeliveryMethod = SmtpDeliveryMethod.Network;
 
-                context.MailMessage.From = new MailAddress(smtpSettings.Address);
+                context.MailMessage.From = new MailAddress(smtpSettings.FromAddress);
```

Comment:
This is an important breaking change, since all existing sites will lose this setting. Maybe a migration step would be good to move the old value to the new properties.

Reply:
@sebastienros  That's a good point, sorry for my mistake. I will continue the work on a migration.
This should be check list item for a contribution note.

Reply:
Update a migration with a test cases result.
However, there is an interesting issue that should fix by other PR.
The existing attribute do not remove from XML after SmtpSettingsPart get updated.
It is still there. I think a serialize XML of saving StmpSettingsPart should be responsible for this.
Therefore, I don't remove it manually.

Could you please check the data after migration in the test case 1.

**Test case 1**
SmtpSettingPart **with existing value** before migration
```
<Data>
    <SiteSettingsPart SiteSalt="fdb3ec855e864eba91d2a3bdfa3c620a" SiteName="orchard-pr" PageTitleSeparator=" - " SiteTimeZone="SE Asia Standard Time" UseFileHash="true" SuperUser="admin" SiteCulture="en-US" PageSize="20" BaseUrl="http://localhost:61745" />
    <CacheSettingsPart VaryByQueryStringIsExclusive="true" />
    <CommentSettingsPart ModerateComments="true" />
    <ThemeSiteSettingsPart CurrentThemeName="TheThemeMachine" />
    <SmtpSettingsPart Address="admin@orchard.com" Host="localhost" Port="25" EnableSsl="false" RequireCredentials="false" UseDefaultCredentials="false" UserName="" Password="" />
</Data>
```

SmtpSettingPart after migration
```
<Data>
    <SiteSettingsPart SiteSalt="fdb3ec855e864eba91d2a3bdfa3c620a" SiteName="orchard-pr" PageTitleSeparator=" - " SiteTimeZone="SE Asia Standard Time" UseFileHash="true" SuperUser="admin" SiteCulture="en-US" PageSize="20" BaseUrl="http://localhost:61745" />
    <CacheSettingsPart VaryByQueryStringIsExclusive="true" />
    <CommentSettingsPart ModerateComments="true" />
    <ThemeSiteSettingsPart CurrentThemeName="TheThemeMachine" />
    <SmtpSettingsPart Address="admin@orchard.com" Host="localhost" Port="25" EnableSsl="false" RequireCredentials="false" UseDefaultCredentials="false" UserName="" Password="" FromAddress="admin@orchard.com" />
</Data>
```

**Test case 2**
SmtpSettingPart **without existing value** before migration 
```
<Data>
    <SiteSettingsPart SiteSalt="fdb3ec855e864eba91d2a3bdfa3c620a" SiteName="orchard-pr" PageTitleSeparator=" - " SiteTimeZone="SE Asia Standard Time" UseFileHash="true" SuperUser="admin" SiteCulture="en-US" PageSize="20" BaseUrl="http://localhost:61745" />
    <CacheSettingsPart VaryByQueryStringIsExclusive="true" />
    <CommentSettingsPart ModerateComments="true" />
    <ThemeSiteSettingsPart CurrentThemeName="TheThemeMachine" />
</Data>
```

SmtpSettingPart after migration
```
<Data>
    <SiteSettingsPart SiteSalt="fdb3ec855e864eba91d2a3bdfa3c620a" SiteName="orchard-pr" PageTitleSeparator=" - " SiteTimeZone="SE Asia Standard Time" UseFileHash="true" SuperUser="admin" SiteCulture="en-US" PageSize="20" BaseUrl="http://localhost:61745" />
    <CacheSettingsPart VaryByQueryStringIsExclusive="true" />
    <CommentSettingsPart ModerateComments="true" />
    <ThemeSiteSettingsPart CurrentThemeName="TheThemeMachine" />
</Data>
```

Here is admin panel after a database migration.
![image](https://user-images.githubusercontent.com/2938310/95839190-ebe1b980-0d6c-11eb-8ee3-557b7631bab8.png)


Sent test email result

![image](https://user-images.githubusercontent.com/2938310/95839233-f7cd7b80-0d6c-11eb-8f01-b5274569ee9d.png)